docker/entrypoint.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+if [ $(id -u qbittorrent) != $CHUID ]; then
+    usermod -o -u $CHUID qbittorrent 
+fi
+
+if [ $(id -g qbittorrent) != $CHGID ]; then
+    groupmod -o -g $CHGID qbittorrent
+fi
+
+# switch to target user
+su qbittorrent
+
+if [ ! -f /config/qBittorrent/qBittorrent.conf ]; then
+    mkdir -p /config/qBittorrent/
+    cat << EOF > /config/qBittorrent/qBittorrent.conf
+[AutoRun]
+enabled=false
+program=
+
+[LegalNotice]
+Accepted=true
```

Comment:
Very thoughtful of you to make it friendly to the users... and give maintainers headaches. :\

Reply:
This is to get webgui out of the box, or it may cannot be accessed(Users cannot type "yes" outof docker).

Reply:
This would be the deal breaker for now, I would object having it accepted by default.
I think WebUI could have a Legal Notice confirmation window to work around this situation.

Reply:
You could add an accept toggle with environment data
`docker run --env LEGAL=accept`